### DISCUSSION TITLE
What size of image input is more suitable

### DISCUSSION THREAD

**Discussion by SCsleepy** [open-mmlab/mmpose#2838] [Q&A] [answered]
Hi,I am using the top-down heatmap+hrnet method to train pose estimation. The image size of my own dataset is 2560 * 1440 (with multiple small target animal cubs), and I have seen that the official configuration file inputs are 256 * 192 and 384 * 288. I would like to ask what size of input is appropriate on my own dataset if the device allows it. If comparative experiments are to be conducted, Which sizes are more suitable for conducting experiments (256 * 192 and 384 * 288 experiments have been completed, and the map for 384 * 288 is higher).

Thank you.

**Comment by Ben-Louis** [comment, marked as answer]
The larger input size generally leads to higher accuracy. As for experiments, using 256 * 192 might be better since it takes a shorter time for each training.
